### ISSUE TITLE
Help me please

### ISSUE THREAD

**Issue by imenbouhdida** [e1560762/rub-conboo-collab#1]
please can u tell me what the 2 files specified in the r.exec() contain? 
the first file project/test/see5/genre and the second one /project/test/see5/rating because i have the error "cannot open file u.names"

**Comment by e1560762** [comment]
Hi,

I developed this system as my term project. I use C5.0[1] clasifier for rule-based approach. I run C5.0 executable from the java program via exec() method. project/test/see5/genre and project/test/see5/rating are relative paths of data files for the existing cases. I suppose your cases and schema are different, so you'd better to replace them with your own files.

For the error "cannot open file u.names", I used MovieLens[2] dataset for user, movie and rating information. u.names is the file for user-related information.

To sum up, I admit that writing the static parameters into the code such a way is a bad approach. A nice solution would be reading those parameters from a configuration file. I developed this system as my term project and the deadline of the project was so close, thus I had to take care of some parts very quickly. It's better to modify related parts of the project or only adopt the required pieces of code.
1. https://www.rulequest.com/see5-info.html
2. http://grouplens.org/datasets/movielens/100k/

**Comment by imenbouhdida** [comment]
Hi, 
Thanks for ur quick response. For now i'm just testing ur version to see how it works so i can modify it. 
I'm just getting blocked with the error "cannot open u.names" so can u please tell me if you remember what was written in your u.names file?
